r/vignettes/developers/install_details.Rmd

```diff
@@ -94,13 +94,14 @@ you'll need to install them before installing the R package.
 
 If libarrow is not found on the system, the R package installation 
 script will next attempt to download prebuilt libarrow binaries
-that match your both your local operating system and arrow R package version.
-The libarrow binaries will only be retrieved if you have set the environment variable
-`LIBARROW_BINARY` or `NOT_CRAN`.
-
-If found, they will be downloaded and bundled when your R package compiles.
-For a list of supported distributions and versions,
-see the [arrow-r-nightly](https://github.com/ursa-labs/arrow-r-nightly/blob/master/README.md) project.
+that match your both your local operating system, required 
+dependencies (e.g. openssl version) and arrow R package version. 
+
+These are used automatically on many Linux distributions (x86_64 architecture only),
+according to the [allowlist](https://github.com/apache/arrow/blob/master/r/tools/nixlibs-allowlist.txt).
+If your distribution isn't in the list, you can opt-in by setting the `NOT_CRAN`
```

Comment:
Could also note here that we welcome PRs to add distros to the allowlist. Although I'm nervous about calling *too* much attention to this feature in the package itself.

Reply:
Yeah same here, that's why I just reused what you had already added in the other vignette, even thought about just not mentioning it xD We can always change it prior to CRAN submission...